packages/carequality-sdk/src/client/carequality.ts

```diff
@@ -97,9 +97,12 @@ export class Carequality {
       throw new BadRequestError(
         `Count value must be between 1 and ${MAX_COUNT}. If you need more, use listAllOrganizations()`
       );
+    const query = new URLSearchParams();
+    oid && query.append("_id", oid);
+    const queryString = query.toString();
+
+    const url = `${Carequality.ORG_ENDPOINT}?apikey=${this.apiKey}&${JSON_FORMAT}&_count=${count}&_start=${start}&${queryString}`;
```

Comment:
I see you added URLSearchParams to populate the `_id` but we should use it for all params of the query 🤓

Reply:
fixed!